svf/include/SVFIR/SVFModuleRW.h

```diff
@@ -195,6 +192,15 @@ class SVFModuleRead
     cJSON* readJson(cJSON* iter, SVFMetadataAsValue* value);
 };
 
+// class SVFIRWriter
```

Comment:
Remove this commented code?

Reply:
Ok. This is commented out because I haven't fully implement it.